app/components/shared/ExecutionStatus.tsx

```diff
@@ -27,108 +26,79 @@ import ExecutionUtils from '../../utils/shared/ExecutionUtils';
  * ```
  *
  */
-export default function ExecutionStatus({
+
+interface Execution {
+    // eslint-disable-next-line camelcase
+    status_display?: string;
+    status?: string;
+    // eslint-disable-next-line camelcase
+    scheduled_for?: string;
+    // eslint-disable-next-line camelcase
+    workflow_id?: string;
```

Comment:
Optional:

Sometimes we use multiple `// eslint-disable-next-line camelcase`, sometimes we close part not using camelcase within `/* eslint-disable camelcase */` and `/* eslint-enable camelcase */` statements. I prefer the second, it seems more clear to me, especially that in backend response there can be more parameters using underscore so it is justified to disable that rule for the whole list.

```suggestion
    /* eslint-disable camelcase */
    status_display?: string;
    status?: string;
    scheduled_for?: string;
    workflow_id?: string;
    /* eslint-enable camelcase */
```

Anyways, it's optional comment.

Reply:
Fixed :wink